src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -23,23 +23,26 @@
      {:color (:button-blue style/colors)})
    :render
    (fn [{:keys [props]}]
-     [:a {:title (:title-text props)
-          :style {:display "inline-block"
-                  :backgroundColor (:color props)
-                  :color "white" :fontWeight 500
-                  :borderRadius 2 :padding (if (:icon props) "0.7em" "0.7em 1em")
-                  :fontFamily (when (:icon props) "fontIcons")
-                  :fontSize (when (:icon props) "80%")
-                  :textDecoration "none"}
-          :href "javascript:;"
-          :onClick (fn [e] ((:onClick props) e))
-          :onKeyDown (common/create-key-handler [:space :enter] (:onClick props))}
-      (or (:text props) (icons/icon-text (:icon props)))
-      (when (= (:style props) :add)
-        [:span {:style {:display "inline-block" :height "1em" :width "1em" :marginLeft "1em"
-                        :position "relative"}}
-         [:span {:style {:position "absolute" :top "-55%" :fontSize "200%" :fontWeight "normal"}}
-          "+"]])])})
+     (let [disabled? (:disabled? props)]
+       [:a {:title (if (and disabled? (string? disabled?)) disabled? (:title-text props))
+            :style {:display "inline-block"
+                    :backgroundColor (:color props)
+                    :WebkitFilter (when disabled? "grayscale()")
+                    :cursor (when disabled? "default")
+                    :color "white" :fontWeight 500
+                    :borderRadius 2 :padding (if (:icon props) "0.7em" "0.7em 1em")
+                    :fontFamily (when (:icon props) "fontIcons")
+                    :fontSize (when (:icon props) "80%")
+                    :textDecoration "none"}
+            :href "javascript:;"
+            :onClick (when-not disabled? (fn [e] ((:onClick props) e)))
```

Comment:
Would prefer a `js/alert` here rather than modifying the title. If I'm clicking frantically on a button because I don't know why it's disabled, I may not hover long-enough to see the reason.